src/extensions/extension-loader.ts

```diff
@@ -332,13 +341,10 @@ export class ExtensionLoader extends Singleton {
   }
 
   toJSON(): Map<LensExtensionId, InstalledExtension> {
-    return toJS(this.extensions, {
-      exportMapsAsObjects: false,
-      recurseEverything: true,
-    });
+    return toJS(this.extensions);
   }
 
-  broadcastExtensions(main = true) {
-    broadcastMessage(main ? ExtensionLoader.extensionsMainChannel : ExtensionLoader.extensionsRendererChannel, Array.from(this.toJSON()));
+  broadcastExtensions(channel = ExtensionLoader.extensionsMainChannel) {
```

Comment:
This shouldn't be an optional parameter.

Reply:
Fixed